application_manager/plugins/spark_mesos/plugin.py

```diff
@@ -0,0 +1,260 @@
+# Copyright (c) 2017 UFCG-LSD and UPV-GRyCAP.
```

Comment:
application_manager/plugins/spark_mesos/plugin.py:23:1: F401 'ID_Generator' imported but unused
from application_manager.utils.ids import ID_Generator
^
application_manager/plugins/spark_mesos/plugin.py:28:1: F401 'json' imported but unused
import json
^
application_manager/plugins/spark_mesos/plugin.py:29:1: F401 'paramiko' imported but unused
import paramiko
^
application_manager/plugins/spark_mesos/plugin.py:73:13: F841 local variable 'actuator' is assigned to but never used
            actuator = str(data['actuator'])
            ^
application_manager/plugins/spark_mesos/plugin.py:75:30: E127 continuation line over-indented for visual indent
                             else int(data['starting_cap'])
                             ^
application_manager/plugins/spark_mesos/plugin.py:77:54: W291 trailing whitespace
            plugin_log.log("%s | Submission id: %s" %
                                                     ^
application_manager/plugins/spark_mesos/plugin.py:84:45: E127 continuation line over-indented for visual indent
                                            api.cluster_username,
                                            ^
application_manager/plugins/spark_mesos/plugin.py:85:45: E127 continuation line over-indented for visual indent
                                            api.cluster_password,
                                            ^
application_manager/plugins/spark_mesos/plugin.py:86:45: E127 continuation line over-indented for visual indent
                                            api.cluster_key_path)
                                            ^
application_manager/plugins/spark_mesos/plugin.py:98:35: E127 continuation line over-indented for visual indent
                                  '--master mesos://%s:%s ' +
                                  ^
application_manager/plugins/spark_mesos/plugin.py:103:35: E127 continuation line over-indented for visual indent
                                  '--master mesos://%s:%s ' +
                                  ^
application_manager/plugins/spark_mesos/plugin.py:113:1: W293 blank line contains whitespace

^
application_manager/plugins/spark_mesos/plugin.py:115:33: E127 continuation line over-indented for visual indent
                                % (time.strftime("%H:%M:%S")))
                                ^
application_manager/plugins/spark_mesos/plugin.py:116:1: W293 blank line contains whitespace

^
application_manager/plugins/spark_mesos/plugin.py:120:33: E127 continuation line over-indented for visual indent
                                % (time.strftime("%H:%M:%S")))
                                ^
application_manager/plugins/spark_mesos/plugin.py:121:1: W293 blank line contains whitespace

^
application_manager/plugins/spark_mesos/plugin.py:124:33: E127 continuation line over-indented for visual indent
                                % (time.strftime("%H:%M:%S")))
                                ^
application_manager/plugins/spark_mesos/plugin.py:139:31: E127 continuation line over-indented for visual indent
                              % (api.one_username,
                              ^
application_manager/plugins/spark_mesos/plugin.py:146:1: W293 blank line contains whitespace

^
application_manager/plugins/spark_mesos/plugin.py:148:60: E127 continuation line over-indented for visual indent
                                                           self.app_id)
                                                           ^
application_manager/plugins/spark_mesos/plugin.py:164:57: E127 continuation line over-indented for visual indent
                                                        api.one_password,
                                                        ^
application_manager/plugins/spark_mesos/plugin.py:165:57: E127 continuation line over-indented for visual indent
                                                        api.one_url)
                                                        ^
application_manager/plugins/spark_mesos/plugin.py:188:80: E501 line too long (80 > 79 characters)
            plugin_log.log("%s | Starting scaler" % (time.strftime("%H:%M:%S")))
                                                                               ^
application_manager/plugins/spark_mesos/plugin.py:206:80: E501 line too long (80 > 79 characters)
            plugin_log.log("%s | Remove binaries" % (time.strftime("%H:%M:%S")))
                                                                               ^
application_manager/plugins/spark_mesos/plugin.py:251:64: E502 the backslash is redundant between brackets
            handling_thread = threading.Thread(target=executor.\
                                                               ^
application_manager/plugins/spark_mesos/plugin.py:259:44: W291 trailing whitespace
        self.running_application = executor


Many pep8 erros

Reply:
Pep8 errors in this file was fixed.